magento/adobe-stock-integration
# Resolved issue #453

### Description

- Added MFTF test as required by issue
- Changed css `position` of the sign in button since the given `z-index` doesn't affect statically positioned elements

### Fixed Issues
- magento/adobe-stock-integration#453: [MFTF] User sees credits available on their account

### Manual testing scenarios
Make sure the ENV file has the following variables
-  `ADOBE_STOCK_USER_EMAIL=<adobe_stock_email_id>`
-  `ADOBE_STOCK_USER_PASSWORD=<adobe_stock_password>`

```bash
$ vendor/bin/mftf run:test AdminAdobeStockSignedInCreditsVisibleTest --remove
```
